components/Newsletter.js

```diff
@@ -44,11 +44,17 @@ const Newsletter = ({ tags, placement }) => {
           { signupError &&
             <div className="error">{ signupError }</div>
           }
-          <div className="flex flex-row items-center justify-center px-2">
-            <input type="email" className="mr-2" value={email} placeholder="Email" onChange={e => setEmail(e.target.value)} required />
-            <button type="submit" name="subscribe" className="btn-primary text-white">
-              Subscribe
-            </button>
+          <div className="flex flex-col items-center justify-start px-2">
+            <p className='mb-2 self-start'>Keep in the loop, subscribe:</p>
+            <input type="email" className="mr-2" value={email} placeholder="Your email" onChange={e => setEmail(e.target.value)} required />
+
+            <div className='flex flex-row justify-between w-80'>
+              <div className='flex flex-row mt-2'>
+              <input type="checkbox"/>
```

Comment:
Let's not show this.

Reply:
Do you mean the input and label that says 'I agree to the...' or the entire div that contains everything?